### PR TITLE
Fix dashboards export test

### LINKED ISSUE #17827: Flaky test: test_dashboard.Test.test_export_dashboard_cmd_export_dashboard_by_id_unknown_id
## Failing Test

* **Test Name:** `test_dashboard.Test.test_export_dashboard_cmd_export_dashboard_by_id_unknown_id`.
* **Link:** https://github.com/elastic/beats/blob/43a07dac7785702129d576dc8a272b4086c84927/libbeat/tests/system/test_dashboard.py#L174

### Stack Trace

```
21:23:56  ======================================================================
21:23:56  FAIL: Test testbeat export dashboard fails gracefully when dashboard with unknown ID is requested
21:23:56  ----------------------------------------------------------------------
21:23:56  Traceback (most recent call last):
21:23:56    File "/go/src/github.com/elastic/beats/libbeat/tests/system/test_dashboard.py", line 191, in test_export_dashboard_cmd_export_dashboard_by_id_unknown_id
21:23:56      assert self.log_contains("error exporting dashboard: Not found") is True
21:23:56  AssertionError
21:23:56  
21:23:56  ----------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by jsoriano**
Test is checking an error message from Kibana that has changed.

Fixes this issue currently affecting master:
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/go/src/github.com/elastic/beats/libbeat/tests/system/test_dashboard.py", line 191, in test_export_dashboard_cmd_export_dashboard_by_id_unknown_id
    assert self.log_contains("error exporting dashboard: Not found") is True
AssertionError
```

Co-Authored-By: Andrew Kroh <andrew.kroh@elastic.co>

Resolves #17827.

### CODE REVIEW

**Comment by jsoriano** [comment]
I have seen that commands tested here print stack traces when fail, I have created another issue as follow up: https://github.com/elastic/beats/issues/17769

File: libbeat/tests/system/test_dashboard.py [hunk 1]
```diff
@@ -183,12 +184,13 @@ def test_export_dashboard_cmd_export_dashboard_by_id_unknown_id(self):
...
                         "-E", "setup.kibana.host=" + self.get_kibana_host(),
                         "-E", "setup.kibana.port=" + self.get_kibana_port(),
-                        "-id", "No-such-dashboard"]
+                        "-id", dashboard_id]
         )
 
         beat.check_wait(exit_code=1)
 
-        assert self.log_contains("error exporting dashboard: Not found") is True
+        expected_error = "error exporting dashboard: Saved object [dashboard/{}] not found".format(dashboard_id)
```
Comment (andrewkroh) [inline libbeat/tests/system/test_dashboard.py:192]: We should probably make it pass on both older and newer versions. Like when running TEST_ENVIRONMENT=latest.

Comment (jsoriano) [reply libbeat/tests/system/test_dashboard.py]: Thanks for the suggestion! Added regexp as in https://github.com/elastic/beats/pull/17773/commits/f6c56fe445eb6ec442e75c2bbb92ec9a2865b408.

**Comment by jsoriano** [comment]
jenkins, test this again

**Comment by jsoriano** [comment]
Errors in CI with kafka images should be solved by https://github.com/elastic/beats/pull/17828

**Comment by jsoriano** [comment]
Trying again after https://github.com/elastic/beats/pull/17819

**Pull request merged**